tls/extensions/s2n_server_sct_list.h

```diff
@@ -15,4 +15,6 @@
 
 #pragma once
 
-extern int s2n_recv_server_sct_list(struct s2n_connection *conn, struct s2n_stuffer *extension);
+int s2n_server_extensions_sct_list_send_size(struct s2n_connection *conn);
```

Comment:
total_size is written as a uint16_t (and just generally should probably be initialized to one). With that in mind, should the return value on your size functions be uint16_t?

Reply:
i kept it in `int` to allow return -1 to represent s2n errors.

Reply:
Our code is all over the place with this convention. In the dev guide it says we should have an output variable in cases like this. Could this function be re-written as `int s2n_server_extensions_sct_list_send_size(struct s2n_connection *conn, uint16_t *out_size);`

Reply:
I'm aware of this, but i'll like to avoid making a bigger extensions_size() function for now, i can address handling the status code instead